SwatDB/SwatDBDataObject.php

```diff
@@ -182,6 +185,12 @@ public function __get($key)
 		if (in_array($key, $this->deprecated_properties))
 			return $this->getDeprecatedProperty($key);
 
+		$property_list = $this->getProtectedPropertyList();
+		if (array_key_exists($key, $property_list)) {
+			$get = $property_list[$key]['get'];
```

Comment:
Need to check if getter exists. If it doesn't exist, should we just return the protected property (default getter)?